x-pack/platform/test/alerting_api_integration/security_and_spaces/group2/tests/alerting/update.ts

```diff
@@ -1497,27 +1496,20 @@ export default function createUpdateTests({ getService }: FtrProviderContext) {
     describe('internally managed rule types', () => {
       const rulePayload = getAlwaysFiringInternalRule();
 
-      const alertUtils = new AlertUtils({
-        user: SuperuserAtSpace1.user,
-        space: SuperuserAtSpace1.space,
-        supertestWithoutAuth,
-      });
-
       it('should throw 400 error when trying to update an internally managed rule type', async () => {
-        const { body: createdRule1 } = await supertest
+        const { body: createdRule } = await supertest
```

Comment:
This test is for `update` but the change was done to `bulk_edit_rules_route`. Should a test also be added to `x-pack/platform/test/alerting_api_integration/security_and_spaces/group3/tests/alerting/bulk_edit.ts`?